File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/NovaPoshta/CityResponse/Datum.java
package com.artmurka.artmurkaapp.Model.Pojo.ItemList.NovaPoshta.CityResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Datum {

    @SerializedName("TotalCount")
    @Expose
    private Integer totalCount;
    @SerializedName("Addresses")
    @Expose
    private List<Address> addresses = null;

    public Integer getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(Integer totalCount) {
        this.totalCount = totalCount;
    }

    public List<Address> getAddresses() {
        return addresses;
    }

    public void setAddresses(List<Address> addresses) {
        this.addresses = addresses;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/novaposhta/CityResponse/Datum.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.CityResponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Datum {

    @SerializedName("TotalCount")
    @Expose
    var totalCount: Int? = null
    @SerializedName("Addresses")
    @Expose
    var addresses: List<Address>? = null

}